CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 ### Bug Fixes
 
+- Skip metrics creation when running the operator locally. ([#2190](https://github.com/operator-framework/operator-sdk/pull/2190))
```

Comment:
We already skip metrics creation when running locally, right? This PR is about logging a more informative message to help the user understand that skipping metrics creation is not an error, right?

Reply:
The current msgs/loggings give an impression that it is an error when is not. 
I improved the description and move to changed section in the changelog.